lib/zendesk_apps_support/package.rb

```diff
@@ -116,40 +116,41 @@ def compile_js(options)
 
       locale = options.fetch(:locale, 'en')
 
-      source = iframe_only? ? nil : app_js
-      version = manifest_json['version']
+      source = manifest.iframe_only? ? nil : app_js
       app_class_name = "app-#{app_id}"
-      author = manifest_json['author']
-      framework_version = manifest_json['frameworkVersion']
-      single_install = manifest_json['singleInstall'] || false
-      signed_urls = manifest_json['signedUrls'] || false
-      templates = is_no_template ? {} : compiled_templates(app_id, asset_url_prefix)
+      # if no_template is an array, we still need the templates
+      templates = manifest.no_template == true ? {} : compiled_templates(app_id, asset_url_prefix)
 
       app_settings = {
-        location: locations,
-        noTemplate: no_template_locations,
-        singleInstall: single_install,
-        signedUrls: signed_urls
+        location: manifest.locations,
+        noTemplate: manifest.no_template_locations,
+        singleInstall: manifest.single_install?,
+        signedUrls: manifest.signed_urls?
       }.select { |_k, v| !v.nil? }
 
       SRC_TEMPLATE.result(
         name: name,
-        version: version,
+        version: manifest.version,
         source: source,
         app_settings: app_settings,
         asset_url_prefix: asset_url_prefix,
         app_class_name: app_class_name,
-        author: author,
+        author: manifest.author,
         translations: runtime_translations(translations_for(locale)),
-        framework_version: framework_version,
+        framework_version: manifest.framework_version,
         templates: templates,
         modules: commonjs_modules,
-        iframe_only: iframe_only?
+        iframe_only: manifest.iframe_only?
       )
     end
 
     def manifest_json
-      @manifest ||= read_json('manifest.json')
+      @manifest_json ||= read_json('manifest.json')
+    end
+    deprecate :manifest_json, :manifest, 2016, 9
```

Comment:
Do we need this? Why not just update all calls to manifest_json and then get rid of it?